hack/unit-test-crds/addontemplates.yaml

```diff
@@ -0,0 +1,472 @@
+# copied from https://github.com/open-cluster-management-io/api/blob/main/addon/v1alpha1/0000_03_addon.open-cluster-management.io_addontemplates.crd.yaml
```

Comment:
Where is this yaml used? I can't find by global search the key "unit-test-crds/addontemplates.yaml"